lexicons/com/atproto/server/describeServer.json

```diff
@@ -9,7 +9,7 @@
         "encoding": "application/json",
         "schema": {
           "type": "object",
-          "required": ["availableUserDomains"],
+          "required": ["did", "availableUserDomains"],
```

Comment:
Ooh this is a sneaky little breaking change we need to be a little careful with for rollout.

Reply:
yup yup  - just need to ensure we update all our PDSs before publishing this new agent library

thought it was important to get in before there are other PDSs out in the wild

Reply:
funny, this bit me too, only just now realized I needed to add it. bsky.app was unhappy with my PDS just now until I did. 😆